src/user-icon/user-icon.scss

```diff
@@ -116,3 +127,54 @@
     }
   }
 }
+
+@mixin iui-user-icon-list {
+  display: flex;
+
+  .iui-user-icon:not(:first-child),
+  > .iui-tooltip-container > .iui-user-icon {
```

Comment:
This is not necessary. Tooltip content can just be placed in the main user list element (after the last user-icon).

Reply:
I've removed line 135 from the CSS and changed the HTML so the last user icon is no longer wrapped with the tooltip container.  I'm not sure I understand how to make the tooltip appear on hover over the last user icon tho.

Reply:
Tooltips are easy in JS (We don't even use `.iui-tooltip-container` in React)

If really want, you could do some CSS hacks here (in demo html), like positioning the tooltip container over the "+5" icon (and giving it some random static width/height). No need to make it perfect.

Reply:
Or you can add this to the end of the html
```html
    <script>
      document.querySelectorAll('#demo-list .iui-user-icon-list').forEach((list) => {
        const lastIcon = list.querySelector('.iui-user-icon:nth-last-child(2)');
        const tooltip = list.querySelector('.iui-tooltip');
        tooltip.style.visibility = 'visible';
        lastIcon.addEventListener('mouseenter', () => { tooltip.style.opacity = 1; });
        lastIcon.addEventListener('mouseleave', () => { tooltip.style.opacity = 0; })
      });
    </script>
```

Reply:
I added it, and it works